modular_skyrat/modules/opposing_force/code/equipment/loadouts.dm

```diff
@@ -0,0 +1,44 @@
+/datum/opposing_force_equipment/loadout
+	category = OPFOR_EQUIPMENT_CATEGORY_LOADOUT
+
+/datum/opposing_force_equipment/loadout/gunman
+	item_type = /obj/item/storage/box/syndie_kit/gunman_outfit
+
+/datum/opposing_force_equipment/loadout/grenadier
```

Comment:
this is really the only thing I'd have an issue with, as it can legitimately crash the server if the whole things goes off at once.

Reply:
That wasn't my addition in the first place, it's been in the game for months already, hasn't seemed to cause any issues yet.